### ISSUE TITLE
pytest-drf breaks `pytest --fixtures`

### ISSUE THREAD

**Issue by last-partizan** [theY4Kman/pytest-drf#3]
Hi

Using ViewSetTest breaks `pytest --fixtures`. Here is minimal reproduction:

```
# test_drf.py
from pytest_drf import ViewSetTest, UsesListEndpoint, Returns200
from pytest_lambda import static_fixture


class Test(ViewSetTest):
    list_url = static_fixture("/test/")

    class TestList(UsesListEndpoint, Returns200):
        pass
```

```
pytest test_drf.py --fixtures

 pytest test_drf.py --fixtures      
platform linux -- Python 3.8.6, pytest-6.0.1, py-1.9.0, pluggy-0.13.1
rootdir: /home/serg/pyright-test
plugins: assert-utils-0.2.1, drf-1.1.0, cases-2.2.1, lambda-1.2.0, fixture-order-0.1.3, common-subject-1.0.4
collected 1 item                                                                                                                    
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/home/serg/.cache/pypoetry/virtualenvs/pyright-test-iQO7wX6m-py3.8/lib/python3.8/site-packages/_pytest/main.py", line 240, in wrap_session
INTERNALERROR>     session.exitstatus = doit(config, session) or 0
INTERNALERROR>   File "/home/serg/.cache/pypoetry/virtualenvs/pyright-test-iQO7wX6m-py3.8/lib/python3.8/site-packages/_pytest/python.py", line 1416, in _showfixtures_main
INTERNALERROR>     available.sort()
INTERNALERROR> TypeError: '<' not supported between instances of 'str' and 'NoneType'
```

It breaks when pytest tries to sort fixtures, becouse `fixturedef.func.__module__` is None for url fixture.

**Comment by theY4Kman** [comment]
Ooooo, good looks, lemme see what I can do

**Comment by theY4Kman** [comment]
Well, I'm still not 100% certain the root cause of this, nor that my fix is the best way to do it, but this is resolved in version 1.1.1 (published to PyPI).

The fix took place in theY4Kman/pytest-lambda, but I added a test for `py.test --fixtures` in pytest-drf. `__module__` was ending up `None` as you said, and I had to apply a fix to two places: 

 1. `error_fixture` used an `exec()` to build a new fixture function from source, but did not curry along the `__module__` attr from its source `error_fn`

 2. Also, we set `self.__module__` in `LambdaFixture` to `parent.__module__` during `contribute_to_parent` (which occurs during collection, after we know the name of the lambda fixture, as well as its parent class/module). `LambdaFixture` itself is a proxy to the resulting fixture function, so setting `self.__module__` _should_ have been enough... but apparently that proved to be false, and an additional `self.fixture_func.__module__ = ...` was required.

`py.test --fixtures` now succeeds for me with your minimal example. Holler if this fix doesn't work for your real case; there might be places I've missed.
